ryaml/__init__.py

```diff
@@ -0,0 +1 @@
+from .ryaml import InvalidYamlError, load, load_all, loads, loads_all, dump, dumps
```

Comment:
Could this file be a `pyi` file as well perhaps?

Reply:
I'm actually thinking we can just omit this line or remove this `__init__.py` file entirely.
I wrote this file based on the `dhall` and `rtoml` packages (like this one: https://github.com/s-zeng/dhall-python/blob/master/dhall/__init__.py) but maybe it's not actually that needed idk

Reply:
I think removing it should be fine, you could move `ryaml.pyi` to `__init__.pyi` and it should work out as users expect I think.

Reply:
The file is already named `__init__.pyi` actually (maybe that's why the import doesn't work uh..)
I'll try something

Reply:
Oh right. The tests are failing because there is no `ryaml.ryaml` to import (`ryaml` lives at the top level). I would move the stub back and remove the `__init__.py`

Reply:
Apparently you can't remove the `__init__.py` file because Maturin need it in order to actually define the whole project as a mixed rust/python package